lib/Pipeline/Context.cpp

```diff
@@ -20,3 +20,42 @@ Logger<> pipeline::CommandLogger("commands");
 
 Context::Context() : TheKindRegistry(Registry::registerAllKinds()) {
 }
+
+llvm::Error Context::store(const revng::DirectoryPath &Path) const {
+  if (auto Error = Globals.store(Path); Error)
+    return Error;
+
+  revng::FilePath IndexPath = Path.getFile(".index");
```

Comment:
The resume directory is already private, let's not use hidden files.

In general, let's never use hidden files unless there's a very good reason to.

Reply:
Done